### PR TITLE
Removed sending null params

### PR DESCRIPTION
**Pull request by F-Ting**
ignore `null` params for term and course

### CODE REVIEW

File: get_info.php [hunk 1]
```diff
@@ -91,12 +91,21 @@ function handle_get($params)
     if (isset($params['target_ta']) && $params['target_ta'] != "null") {
         $bind_variables[':target_ta'] = $params['target_ta'];
     }
-    if (isset($params["term"]) && $params['term'] != "null") {
+    if (isset($params["term"])) {
+      if($params['course_code'] != "null") {
         $bind_variables[':term'] = $params["term"];
+      }else{
```
Comment (siefkenj) [inline get_info.php]: prettier

File: get_info.php [hunk 2]
```diff
@@ -91,12 +91,21 @@ function handle_get($params)
...
+        unset($params["term"]);
```
Comment (siefkenj) [inline get_info.php]: I don't like this. Look at `santitize_arguments` from `post_info.php`

File: get_info.php [hunk 3]
```diff
@@ -168,7 +177,7 @@ function handle_get($params)
             }
             $user_id = $params["user_id"];
             $target_ta = $params["target_ta"];
-            $course_code = array_get($params, "course");
+            $course_code = array_get($params, "course_code");
```
Comment (siefkenj) [inline get_info.php]: This is wrong. The API specifies `"course"`

Comment (F-Ting) [reply get_info.php]: There is a inconsistency in the API documentation, I assumed it to be `course_code` for everything. Thus all API requests have `course_code` in the front-end. Could I change the API specification to be `course_code`?

Comment (siefkenj) [reply get_info.php]: You may. You will also need to change the test that uses `"course"`

**Pull request merged**